.github/workflows/build.yml

```diff
@@ -208,6 +208,11 @@ jobs:
             ##########
             # Caches #
             ##########
+
+            # needed for act
```

Comment:
I would avoid these

Reply:
`act` compatibility would be really nice to have for sanity-checking changes to CI - it's hard maintaining a state clean enough locally to replicate CI issues, and pushing to github actions is sloooow

that said it's likely that just giving in and using one of the bigger images would work here

Reply:
using one of the fat images seems to work. my ~/.actrc

```
--container-architecture linux/amd64
-P ubuntu-20.04=catthehacker/ubuntu:full-20.04
```